src/lib/GridItem.svelte

```diff
@@ -122,20 +122,71 @@
 		left = _left;
 		top = _top;
 
-		window.scroll({
-			left: left - window.innerWidth / 2,
-			top: top - window.innerHeight / 2,
-			behavior: 'smooth'
-		});
+		if (!collision) {
+			window.scroll({
+				left: left - window.innerWidth / 2,
+				top: top - window.innerHeight / 2,
+				behavior: 'smooth'
+			});
+		}
```

Comment:
Stretching the scroll with moving or resizing triggers bugs like trembling or reverse scrolling. Collision check is to prevent conflicts between your code and mine.